BUILD.md

```diff
@@ -104,6 +104,87 @@ The following dependencies are required for this project:
     ctest --test-dir build --output-log build
     ```
 
+## Compilation steps for Windows
+
+1. **Installing Dependencies**
+
+- Microsoft Visual C++ (MSVC) 14.
+- Chocolatey
+    ```bash
+    Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+    ```
+- Make
+    ```bash
+    choco insall make
+    ```
+- Cmake 3.30.3
+    ```bash
+    choco insall cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+    ```
+- LLVM (LLVM-18.1.8-win64.exe) from https://github.com/llvm/llvm-project/releases/tag/llvmorg-18.1.8
+- Git
+
+2. **Clone the Repository**
+
+    First, clone the repository using the following command:
+
+    ```bash
+    git clone https://github.com/wazuh/wazuh-agent.git
+    ```
+
+3. **Initialize Submodules**
+
+    The project uses submodules, so you need to initialize and update them. Run the following commands:
+
+    ```bash
+    cd wazuh-agent
+    git submodule update --init --recursive
+    ```
+
+4. **Configure and Build the Project**
+
+    ```bash
+    cmake src -B build
+    cmake --build build
+    ```
+
+    If you want to include tests, configure the project with the following command:
+
+    ```bash
+    cmake src -B build -DBUILD_TESTS=1 -G "Visual Studio 17 2022" -A x64
+    cmake --build . --config RelWithDebInfo
```

Comment:
Should be `cmake --build build --config RelWithDebInfo`

Reply:
Fixed!